### PR TITLE
[Doc] Add link to paper in readme

### PR DESCRIPTION
**Pull request by giadefa**
Added link to paper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ TorchRL aims at having as few dependencies as possible (python standard library,
 
 On the low-level end, torchrl comes with a set of highly re-usable functionals for [cost functions](torchrl/objectives/costs), [returns](torchrl/objectives/returns) and data processing.
 
-TorchRL aims at (1) a high modularity and (2) good runtime performance.
+TorchRL aims at (1) a high modularity and (2) good runtime performance. Read the full paper here: https://arxiv.org/abs/2306.00577.
```
Comment (vmoens) [inline README.md:33]: why not 
```suggestion
TorchRL aims at (1) a high modularity and (2) good runtime performance. Read the [full paper](https://arxiv.org/abs/2306.00577) for a more curated description of the library.
```
or something similar?

**Review by vmoens — APPROVED** [review]
Just an aesthetic suggestion to avoid having the plain link exposed in the text.

**Comment by matteobettini** [comment]
Just a note. Check this doesn't show up in the anonymized version after merging this

**Comment by matteobettini** [comment]
If all links are blocked in the anonymized github link we can merge this without problems.

**Pull request merged**